iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/VscConverterStationModificationTest.java

```diff
@@ -54,8 +54,7 @@ void testApplyChecks() {
             2.);
         assertThrows(PowsyblException.class, () -> modif1.apply(network, true, ReportNode.NO_OP),
             "An invalid ID should fail to apply.");
-        assertDoesNotThrow(() -> modif1.apply(network, false, ReportNode.NO_OP),
-            "An invalid ID should not throw if throwException is false.");
+        assertDoesNotThrow(() -> modif1.apply(network, false, ReportNode.NO_OP));
```

Comment:
Can you revert this change?
The message is legit here: it corresponds to the behavior when the assertion fails.